### ISSUE TITLE
Lower validation accuracy

### ISSUE THREAD

**Issue by monaj07** [speedinghzl/Pytorch-Deeplab#7]
First of all, thanks for providing your code.
When I use your uploaded `VOC12_scenes_20000.pth` model to evaluate the single scale model on 1449 validation data, I get 71.8% which is much lower than reported 74.0%.
Moreover, when I try to train the single scale model by running `python train.py --random-mirror --random-scale --gpu 0`, and then evaluate the trained model, I get 73.1%, which is still lower than 74%.
Do you know where this discrepancy might come from?
Everything is on default settings and I also use `MS_DeepLab_resnet_pretrained_COCO_init.pth` as the pre-trained model.

**Comment by speedinghzl** [comment]
Thanks. I will check the performance of VOC12_scenes_20000.pth and I suggest you try more than once to train single-scale deeplab.

**Comment by monaj07** [comment]
Thanks, sure, I repeat the training to see if it changes the results.
Have you possibly set a manual seed for getting reproducible results?
Please let me know the outcome of your performance evaluation using your uploaded `VOC12_scenes_20000.pth`. Thanks.

**Comment by speedinghzl** [comment]
I get 73.8 mean IOU on val set with VOC12_scenes_20000.pth. It's great to set a manual seed. I will add it later.

**Comment by monaj07** [comment]
I see, thanks for reporting back.
So what do you think the problem might be?
In `evaluate.py`, I simply just modified lines 22 and 27 for the path to the data as well as the path to VOC12_scenes_20000.pth.
Everything else is the same and I am a bit confused.

**Comment by sinemaslanx** [comment]
Could anybody share VOC12_scenes_20000.pth on googledrive? I couldnot install it from the link given here: https://pan.baidu.com/s/1bP52R8

in chinese and it requires me to install a exe file.. baudi..something ..
